src/debug_info.rs

```diff
@@ -107,11 +107,7 @@ impl<'c> DebugLocations<'c> {
             .map(|(statement_idx, location_id)| {
                 (
                     *statement_idx,
-                    extract_location_from_stable_loc(
-                        context,
-                        db,
-                        location_id.get(db).stable_location,
-                    ),
+                    extract_location_from_stable_loc(context, db, location_id.all_locations(db)[0]),
```

Comment:
Are we sure this line is fine? I indexed it to 0 at that time in order to be able to compile. Due to the fact that now there's not just one stable location but maybe more (because it fetches `all_locations`) I don't know if it could bring issues in the future, and that could be difficult to find. Perhaps it is worth to take a look at it before merging.